components/ServiceDetails/Sections/Info.vue

```diff
@@ -51,7 +51,7 @@ export default {
   
   computed: {
     deployCommand() {
-      return `mesg:marketplace:service:${this.sid}:${this.lastVersion}`
+      return `mesg-core service deploy mesg://marketplace/service/${this.lastVersion}`
```

Comment:
```suggestion
      return `mesg://marketplace/service/${this.lastVersion}`
```

Reply:
actually the previous code wrong, I fixed and wouldn't update this line. Otherwise clipboard copy will only copy the link and not the full command. Please tell me if you think that copying only the url is the better option

Reply:
ok so maybe need some update on the UI
<img width="461" alt="screen shot 2019-02-20 at 10 42 27 am" src="https://user-images.githubusercontent.com/1783126/53064957-44ab7100-34fc-11e9-8f8f-1ecdcdd9338b.png">
there is two times `mesg-core service deploy`

Reply:
https://github.com/mesg-foundation/marketplace.mesg.com/pull/3/commits/9740d20bec30d32913eec56e988e6631fd77c6fc